deployment/docker/auth-vl-single.yml

```diff
@@ -0,0 +1,12 @@
+# route requests between VictoriaMetrics and VictoriaLogs
+unauthorized_user:
```

Comment:
why the previous config https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/auth-mixed-datasource.yml didn't work? It seems easier to me, as it doesn't drop prefix parts.

Reply:
Had to remove it, as it didn't work for vmalert querying two different datasources.